src/details/ArborX_DetailsUtils.hpp

```diff
@@ -585,6 +586,38 @@ template <typename View>
   return clone(ExecutionSpace{}, v);
 }
 
+template <typename ExecutionSpace, typename View, typename Offset>
+void computeOffsetsInOrderedView(ExecutionSpace const &exec_space, View view,
```

Comment:
What do you think of
```suggestion
void computeOffsets(ExecutionSpace const &space, OrderedView view,
```

Reply:
I'm of two minds here. The function produce output even if it's given a view of random numbers. For example, `{2, 3, 2, 3}` would produce `{0, 1, 2, 3, 4}`. But I don't see a situation where that is useful. With a `InOrderedView` I want a caller to take a second to think to catch a potential mistake: is the view I'm passing in ordered?